app/models/concerns/foreman_ansible/host_managed_extensions.rb

```diff
@@ -67,6 +67,11 @@ def import_host(*args)
 
 module Host
   class Managed
+    apipie :class do
```

Comment:
Curiously enough, the following three do not show up. Is it just me? Changes to ansible_role and renderer_methods work as expected.

Reply:
I do see them.. 
![ScreenShot-1594815388668](https://user-images.githubusercontent.com/32508194/87543898-d42f6200-c6a5-11ea-939d-88459143d892.png)

They are added to `Host Managed` class as properties. Have you re-cached docs?

Reply:
I see them too